src/stan_math_backend/Stan_math_code_gen.ml

```diff
@@ -376,8 +381,10 @@ let pp_ctor ppf (p : typed_prog) =
   in
   let rec pp_statement_zeroing ppf s =
     match s.stmt with
-    | Decl {decl_id; decl_type; _} ->
-        pp_set_size ppf (decl_id, decl_type, DataOnly)
+    | Decl {decl_id; decl_type; _} -> (
+      match decl_type with
+      | Sized st -> pp_set_size ppf (decl_id, st, DataOnly)
+      | Unsized _ -> () )
```

Comment:
Should this throw an exception? Or ... idk, somehow () seems unlikely to be correct, but not sure